File: Q11 find all duplicates in an array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var findDuplicates = function(nums) {
    nums.sort((a, b) => a - b); 
    var a = [];
    for(var i=0;i<nums.length-1;i++){
        if(nums[i]==nums[i+1]){
            a.push(nums[i]);
        }
    }
    return a;
};

File: Q11 find all duplicates in an array.py
class Solution(object):
    def findDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        nums.sort()
        my_list = []
        for i in range (len(nums)-1):
            if(nums[i]==nums[i+1]):
              my_list.append(nums[i])
        return my_list     

        
